src/components/Landing.js

```diff
@@ -2,35 +2,68 @@ import React from "react";
+import LandingCard from "./Landing/LandingCard";
+// import { Snowfall } from "react-snowfall";
 
 function Landing() {
-    const heroTitle = "ACM Admin Portal"
-    const heroSubtitle = "Unleash efficiency, foster excellence, and streamline administrative tasks effortlessly";
+  return (
+    <div className="grid lg:grid-cols-2 grid-cols-1 h-[100vh] bg-[#edf3ff]">
+      {/* <Snowfall
+        color="#9ab4d1"
+        wind={[0, 0]}
+        snowflakeCount={20}
+        speed={[0.1, 2.5]}
+        radius={[1, 5]}
+      /> */}
 
-    return (
-        <div className="grid lg:grid-cols-2 grid-cols-1 h-[100vh]">
-            <Head>
-                <title>Admin Portal | PECACM</title>
-            </Head>
-            <div className="bg-[#d9d9d9] h-[100vh] justify-center items-center hidden lg:grid">
-                <img src="/assets/acm.png" height={500} width={500} />
-            </div>
+      <Head>
+        <title>Admin Portal | PECACM</title>
+      </Head>
+      <div className="min-w-[100vw] flex flex-col justify-center items-center">
+        <img
+          src="/assets/acm.png"
+          height={200}
+          width={200}
+          className="mr-[10px]"
+        />
+        <p className="flex justify-center items-center text-[70px] text-[#3d6ca3] font-[700]">
+          PEC ACM Admin Portal
+        </p>
 
-            <div className="lg:h-[100vh]">
-                <div className="p-2 lg:hidden">
-                    <img src="/assets/acm.png" height={80} width={80} />
-                </div>
+        <p className="text-[#3d6ca3] text-[26px] font-normal opacity-60 mt-5">
+          Powering Student Excellence and Efficiency
+        </p>
 
-                <div className="justify-center items-center flex flex-col mt-10 lg:mt-0 lg:min-h-[100vh]">
-                    <p className="text-5xl md:text-8xl font-bold text-[#1d4ed8]">Welcome</p>
-                    <HeroBox title={heroTitle} subtitle={heroSubtitle} />
-                    <Link href={"/login"} aria-label="Login">
-                        <button className="acm-button px-[40px]">Log In</button>
-                    </Link>
-                </div>
-            </div>
+        <div className="flex my-10">
+          {LANDINGCARDS_DATA.map(({ imgPath, title, link }, id) => (
+            <LandingCard imgPath={imgPath} title={title} link={link} key={id} />
+          ))}
         </div>
-    );
+
+        <Link href={"/login"} aria-label="Login">
+          <button className="acm-button px-[40px]">Log In</button>
+        </Link>
+      </div>
+    </div>
+  );
 }
 
 export default Landing;
+
+const LANDINGCARDS_DATA = [
+  {
+    imgPath: "/assets/acm.png",
+    title: "Meet our team",
+    link: "https://www.pecacm.com/team",
+  },
+  {
+    imgPath: "/assets/acm.png",
+    title: "Meet our team",
+    link: "https://www.pecacm.com/team",
+  },
+  {
+    imgPath: "/assets/acm.png",
+    title: "Meet our team",
+    link: "https://www.pecacm.com/team",
+  },
```

Comment:
These items can be
- Meet our team (some icon of team)
- Continue to portal (pecacm.com) -- icon of pecacm
- Checkout leaderboard (pecacm.com/dashboard/leaderboard)